go.mod

```diff
@@ -119,4 +121,6 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.7.0 // indirect
 )
 
+replace github.com/kr/pretty => github.com/kr/pretty v0.2.1
```

Comment:
do we need to do the replace here? can we just use v0.2.1 at the higher level instead or is there a reason to have it at version v0.3.1?